tests/goss.clamav.yml

```diff
@@ -0,0 +1,41 @@
+package:
+  clamav-daemon:
+    installed: true
+    versions:
+    - 0.102.4-r1
+  clamav-libunrar:
+    installed: true
+    versions:
+    - 0.102.4-r1
+port:
+  tcp:3310:
+    listening: true
+    ip:
+    - 0.0.0.0
+process:
+  clamd:
+    running: true
+  freshclam:
+    running: true
+file:
+  /etc/clamav/clamd.conf:
+    exists: true
+    mode: "0644"
+    owner: root
+    group: root
+    filetype: file
+    contains:
+    - '# Debug yes'
+    - LogSyslog yes
+    - '# LogFile /dev/stdout'
+    - StreamMaxLength 260M
+    - Foreground yes
+    - MaxScanSize 256M
+    - MaxFileSize 256M
+  /etc/clamav/freshclam.conf:
+    exists: true
+    mode: "0644"
+    owner: root
+    group: root
+    filetype: file
+    contains: []
```

Comment:
@steveworley this is kinda what I was thinking - if this is possible it would keep the test entrypoint in one spot, easier to run locally etc..

```
command:
  "cd /app/clamav && ./vendor/bin/phpunit":
    exit-status: 0
```

Reply:
Yeah agreed - the only issue with this is that the PHPunit stuff isn't copied into the clam container its added in a separate container and makes a TCP request into a running clamav container on the same docker network.

Add a goss test for `goss.test.yaml` to run this and we always have an exit status of 2.

```
Command: cd /app/clamav && ./vendor/bin/phpunit: exit-status:
Expected
    <int>: 2
to equal
    <int>: 0
```

From the looks it's because `dgoss` only runs on a single image to validate its configuration. Even with the clam container running locally this doesn't work quite the same way as a docker-compose exec. I tried setting `CONTAINER_LOG_OUTPUT` to see what was happening but the log stream was always empty.

The same kind of thing happens with testing nginx config (which is why I removed it here) - it is expecting the PHP container to be available as thats the server when running goss against that:

```
nginx: [emerg] host not found in upstream "php" in /etc/nginx/conf.d/app.conf:67
ERROR: the container failed to star
```

Reply:
Ah ok fair enough, I didnt notice it was in a different container to clam. Carry on!